src/examples/pagination/CursorDefault.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { CursorPagination } from '@zendeskgarden/react-pagination';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => {
+  const [cursor, setCursor] = useState(0);
+
+  const pages = [0, 1, 2, 3, 4];
+
+  const onFirst = () => setCursor(0);
+
+  const onLast = () => setCursor(pages.length - 1);
+
+  const onNext = () => {
+    if (cursor < pages.length - 1) {
+      setCursor(cursor + 1);
+    }
+  };
+
+  const onPrevious = () => {
+    if (cursor > 0) {
+      setCursor(cursor - 1);
+    }
+  };
+
+  return (
+    <Row alignItems="center">
+      <Col alignSelf="center">
+        <CursorPagination aria-label="Cursor pagination">
+          <CursorPagination.First onClick={onFirst} disabled={cursor === 0}>
+            First
+          </CursorPagination.First>
+          <CursorPagination.Previous onClick={onPrevious} disabled={cursor === 0}>
+            Previous
+          </CursorPagination.Previous>
+          <CursorPagination.Next onClick={onNext} disabled={cursor === pages.length - 1}>
```

Comment:
I pulled this example code from the existing site but build is failing on this disabled attribute. Typescript doesn't like it and I wasn't sure how to fix it on my end. Could use some guidance.

Reply:
I think this could be a type definition issue unrelated to this PR (rather an issue in `react-components`). Looking into it now 🤔 .

Reply:
Looks like the current type definition can be more specific than `HTMLAttributes`. The `disabled` attribute isn't a part of `HTMLAttributes`. I'll go ahead and open a fix for this in `react-components` that uses `ButtonHTMLAttributes` instead 👍 .